internal/controller/storageclient_controller.go

```diff
@@ -283,6 +283,28 @@ func (r *StorageClientReconciler) reconcilePhases() (ctrl.Result, error) {
 			if err != nil {
 				return reconcile.Result{}, fmt.Errorf("failed to create remote noobaa: %v", err)
 			}
+		case "ConfigMap":
+			configMap := &corev1.ConfigMap{}
+			configMap.Name = eResource.Name
+			configMap.Namespace = r.OperatorNamespace
+
+			data := map[string]string{}
+			err = json.Unmarshal(eResource.Data, &data)
+			if err != nil {
+				return reconcile.Result{}, fmt.Errorf("failed to unmarshal %s configuration response: %v", eResource.Name, err)
+			}
+
+			_, err := controllerutil.CreateOrUpdate(r.ctx, r.Client, configMap, func() error {
```

Comment:
in think we have internal `createOrUpdate` can we use it?

Reply:
done